app/models/occupation_standard.rb

```diff
@@ -218,6 +219,8 @@ def as_indexed_json(_ = {})
 
   scope :with_work_processes, -> { joins(:work_processes).distinct }
 
+  scope :recently_added, -> { order(created_at: :desc).limit(MAX_RECENTLY_ADDED_OCCUPATIONS_TO_DISPLAY) }
```

Comment:
@FerPerales I think we still need to hook into ES here, or otherwise skip occupation standards that have no work processes ([per this constraint](https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/blob/main/app/queries/occupation_standard_elasticsearch_query.rb#L29-L31)) if we want to have a postgres query here. It seems a little overkill to go to ES for this; on the other hand, it will make sure our queries are consistent if we add other constraints. I will defer to you.

Reply:
@littleforest I updated the scope to exclude occupation standards without work processes using [associated](https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods/WhereChain.html#method-i-associated)